docs/api/ReactWrapper/invoke.md

```diff
@@ -1,6 +1,8 @@
 # `.invoke(propName)(...args) => Any`
 
 Invokes a function prop.
+Note that `.invoke` will wrap your handler with [`ReactTestUtils.act`](https://reactjs.org/docs/test-utils.html#act) and call `.update()` automatically. It would
+be less error prone than directly call handler prop with `.props()`.
```

Comment:
```suggestion
```

Reply:
IMO,
```javascript
wrapper.invoke('handler')
expect(...)
```
is better than
```javascript
ReactTestUtils.act(wrapper.prop('handler')());
wrapper.update();
expect(...)
```
if we simply want to trigger handler. Since it's easy to forget (1) the `act` wrap and (2) the `update` call. I think this knowledge may worth a document (or maybe my opinion is wrong).

Reply:
I agree; I folded this line into the one above it.